src/main/java/org/apache/maven/report/projectinfo/DependenciesReport.java

```diff
@@ -205,45 +205,27 @@ private DependencyNode resolveProject() {
      * @throws IOException if any
      */
     private void copyResources(File outputDirectory) throws IOException {
-        InputStream resourceList = null;
-        InputStream in = null;
-        BufferedReader reader = null;
-        OutputStream out = null;
-        try {
-            resourceList = getClass().getClassLoader().getResourceAsStream(RESOURCES_DIR + "/resources.txt");
-
-            if (resourceList != null) {
-                reader = new LineNumberReader(new InputStreamReader(resourceList, ReaderFactory.US_ASCII));
-
+        InputStream resourceList = getClass().getClassLoader().getResourceAsStream(RESOURCES_DIR + "/resources.txt");
```

Comment:
`InputStream` should be in try

Reply:
I'm not sure how to do the null check if I put it there. Closing the wrapping reader should close the input stream if needed.